src/panels/config/integrations/integration-panels/dynalite/dynalite-config-panel.ts

```diff
@@ -0,0 +1,279 @@
+				<span slot="description">${this.localStr("port_long")}</span>
+                <paper-input
+                  class="flex"
+                  .label=${this.localStr("port")}
+                  name="port"
+                  type="number"
+                  .value=${this._port}
+                  @value-changed=${this._handleChange}
+                >
+			  </ha-settings-row>
+            </div>
+            <div class="card-content">
+			  <ha-settings-row .narrow=${this.narrow}>
+				<span slot="heading">${this.localStr("fade")}</span>
+				<span slot="description">${this.localStr("fade_long")}</span>
+                <paper-input
+                  class="flex"
+                  .label=${this.localStr("fade")}
+                  name="fade"
+                  type="number"
+                  .value=${this._fade}
+                  @value-changed=${this._handleChange}
+                >
+			  </ha-settings-row>
+            </div>
+            <div class="card-content">
+			  <ha-settings-row .narrow=${this.narrow}>
+				<span slot="heading">${this.localStr("active")}</span>
+				<span slot="description">${this.localStr("active_long")}</span>
+				<ha-paper-dropdown-menu label=${this.localStr("active")} dynamic-align>
+				  <paper-listbox
+					slot="dropdown-content"
+					selected=${this._activeIndex}
+                    @iron-select=${this._handleActiveSelection}
+				  >
+					${this._activeOptions.map(
+            (option) =>
+              html`<paper-item .active_config=${option}
+                >${this.localStr("active_" + option)}</paper-item
+              >`
+          )}
+				  </paper-listbox>
+				</ha-paper-dropdown-menu>
+			  </ha-settings-row>
+            </div>
+            <div class="card-content">
+			  <ha-settings-row .narrow=${this.narrow}>
+				<span slot="heading">${this.localStr("auto_discover")}</span>
+				<span slot="description">${this.localStr("auto_discover_long")}</span>
+				<ha-switch
+				  .checked=${this._auto_discover}
+				  @change=${this._handleAutoDiscoverChange}
+				></ha-switch>
+			  </ha-settings-row>
+            </div>
+            <div class="card-content">
+			  <ha-settings-row .narrow=${this.narrow}>
+				<span slot="heading">${this.localStr("poll_timer")}</span>
+				<span slot="description">${this.localStr("poll_timer_long")}</span>
+                <paper-input
+                  class="flex"
+                  .label=${this.localStr("poll_timer")}
+                  name="poll_timer"
+                  type="number"
+                  .value=${this._poll_timer}
+                  @value-changed=${this._handleChange}
+                >
+			  </ha-settings-row>
+            </div>
+            <div class="card-actions">
+              <mwc-button @click=${this._publish}>
+			    ${this.localStr("publish")}
+			  </mwc-button>
+            </div>
+          </ha-card>
+        </div>
+      </ha-app-layout>
+    `;
+  }
+
+  private localStr(item) {
```

Comment:
Private functions should be lower than public or protected

Also Idk if this is how we want to do this. I know it make it easier and you are using all the same localize. but we don't do this any where else. Up to @bramkragten

Reply:
what do you mean lower?

Reply:
Lower in the code. private function should be under public and protected function. So if at line 5 then it should be lower aka line 10

Reply:
ok. i assume lower in the code. pushed an update with it